isc26-EESSI/main.tex

```diff
@@ -0,0 +1,183 @@
+\documentclass[a4paper,11pt]{article}
+
+% Allow coloured url links and hyperlinked table of contents etc....
+\usepackage[colorlinks=true, linkcolor=gray,  urlcolor=blue, citecolor=black]{hyperref}
+\usepackage{authblk}
+\usepackage{todonotes}
+
+\usepackage{enumitem}
+\setlist{itemsep=2pt, parsep=2pt}
+
+\setlength{\textwidth}{6.5in}
+\setlength{\textheight}{8.6in}
+\setlength{\topmargin}{0.0in}
+\setlength{\headheight}{0.0in}
+\setlength{\oddsidemargin}{0.0in}
+\setlength{\evensidemargin}{0.0in}
+
+\newcommand{\ignore}[1]{}
+
+\newif\ifremark
+\long\def\remark#1{
+\ifremark%
+        \begingroup%
+        \dimen0=\columnwidth
+        \advance\dimen0 by -1in%
+        \setbox0=\hbox{\parbox[b]{\dimen0}{\protect\em #1}}
+        \dimen1=\ht0\advance\dimen1 by 2pt%
+        \dimen2=\dp0\advance\dimen2 by 2pt%
+        \vskip 0.25pt%
+        \hbox to \columnwidth{%
+                \vrule height\dimen1 width 3pt depth\dimen2%
+                \hss\copy0\hss%
+                \vrule height\dimen1 width 3pt depth\dimen2%
+        }%
+        \endgroup%
+\fi}
+
+\remarktrue
+%%\remarkfalse
+
+\newenvironment{myenumerate}
+{ \begin{enumerate}
+    \setlength{\itemsep}{0pt}
+    \setlength{\parskip}{0pt}
+    \setlength{\parsep}{0pt}     }
+{ \end{enumerate}                  } 
+
+\renewcommand{\baselinestretch}{0.95}
+\newcommand{\comment}[1]{\todo[inline]{#1}}
+
+
+\begin{document}
+
+%===============================================================================
+
+\title{
+    \textbf{\LARGE Streaming Optimised Scientific Software: an Introduction to EESSI}\\
+\vspace{2mm}{\Large \emph{ISC'24 tutorial proposal}}
+}
+
+\date{}
+
+\author[1]{Sebastian Achilles}
+\author[2]{Kenneth Hoste}
+\author[3]{Alan \'O Cais}
+\affil[1]{J\"ulich Supercomputing Centre, Forschungszentrum J\"ulich GmbH, Germany}
+\affil[2]{HPC-UGent, Ghent University, Belgium}
+\affil[3]{Centre Européen de Calcul Atomique et Moléculaire (CECAM), Switzerland}
```

Comment:
```suggestion
\author[1]{Kenneth Hoste}
\author[1]{Lara Peeters}
\author[3]{Helena Vela Beltran}
\author[2]{Sebastian Achilles}
\author[1]{Alan \'O Cais}
\affil[1]{HPC-UGent, Ghent University, Belgium}
\affil[2]{J\"ulich Supercomputing Centre, Forschungszentrum J\"ulich GmbH, Germany}
\affil[3]{Do IT Now Spain}
```

@SebastianAchilles Do let us know if you're happy to be involved again

@hvelab Please check affiliation